README.md

```diff
@@ -92,6 +92,16 @@ public class MainActivity extends AppCompatActivity {
 
 ```
 
+Note for using in Android library 
+---------------------------------
+In case you develop an Android library and use this repository, add this into the dependencies:
```

Comment:
As I already mentioned in #45 this dependency is added automatically by the plugin so you don't have to add it yourself when you develop a library or an app. Moreover, adding this dependency doesn't mean it's going to be transitively added to a project that uses this library. To make it happen you have to publish the library with a proper pom.xml, which isn't related to Paranoid.

Reply:
I don't understand when is it needed, then, if using the plugin already takes care of it. 
And why did you want me to have a pull request to write something that is wrong...

Reply:
I tried to explain that what you suggest isn't related to the library and asked you to continue the discussion in some other place rather than in an irrelevant issue.

The dependency is needed when you distribute a library incorrectly. And that's an issue with the distribution process, which isn't related to this library.

Actually, there's a case when adding the dependency manually is needed and it happens in case of multi-module projects. But it's a very different topic.

Reply:
I see. I'm very sorry for all the trouble, and in fact I do have an issue with publishing properly. If you wish, I wrote a new question on StackOverflow about it, [here](https://stackoverflow.com/q/66983761/878126), with a bounty. If you succeed solving it, I will grant you the bounty.

Reply:
I answered your question. Hope it'll help.